src/styles/index.scss

```diff
@@ -1,3 +1,235 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap';
```

Comment:
Please remove the import URL or replace it with a valid path. Import statements should refer to local files or packages.